.github/workflows/publish_docker_image.yml

```diff
@@ -20,3 +20,4 @@ jobs:
           DOCKER_PASS: ${{ secrets.docker_pass }}
           DOCKER_REPO: ${{ secrets.docker_repo }}
         run: .github/workflows/publish_image.sh
+// TODO@ pure avax copy, do we need it?
```

Comment:
IMO no -> remove

Reply:
thats outdated, was addressed in some of fixes commits